aion/web/elm/src/Update.elm

```diff
@@ -121,32 +140,32 @@ update msg model =
             else
                 model ! []
 
-        NoOperation ->
-            model ! []
-
-        SetNewQuestionContent questionContent ->
+        UpdateCreateQuestionForm name value ->
             let
                 oldPanelData =
                     model.panelData
             in
-                { model | panelData = { oldPanelData | newQuestionContent = questionContent } } ! []
+                { model
+                    | panelData =
+                        { oldPanelData | createQuestionForm = Forms.updateFormInput oldPanelData.createQuestionForm name value }
+                }
+                    ! []
 
-        SetNewAnswerContent answerContent ->
+        CreateNewQuestionWithAnswers ->
             let
-                oldPanelData =
-                    model.panelData
+                possibleFields =
+                    [ "question", "answers", "subject" ]
+
+                validationErrors =
+                    possibleFields
+                        |> List.map (\name -> Forms.errorList model.panelData.createQuestionForm name)
+                        |> List.foldr (++) []
+                        |> List.filter (\validations -> validations /= Nothing)
             in
-                { model | panelData = { oldPanelData | newAnswerContent = answerContent } } ! []
-
-        SetNewAnswerCategory answerCategoryName ->
-            let
-                answerCategoryToId =
-                    getSubjectIdByName model.rooms answerCategoryName
-
-                oldPanelData =
-                    model.panelData
-            in
-                { model | panelData = { oldPanelData | newAnswerCategory = answerCategoryToId } } ! []
+                if List.isEmpty validationErrors then
+                    ( model, createQuestionWithAnswers model )
+                else
+                    model ! []
```

Comment:
What happens if there actually are errors? (serious question, what do we want to do, react somehow - flash info message or just leave it and let user figure out why it's not working?)